inode/config.go

```diff
@@ -192,6 +199,9 @@ func Up(confMap conf.ConfMap) (err error) {
 	globals.accountMap = make(map[string]*volumeStruct)
 	globals.flowControlMap = make(map[string]*flowControlStruct)
 
+	var tmpInode inMemoryInodeStruct
+	globals.inodeSize = uint64(unsafe.Sizeof(tmpInode))
```

Comment:
See my earlier comment.

If you look at the inMemoryInodeStruct, you will see all manner of by-reference fields.
So I guess I shouldn't be surprised that the value calculated here is 300 Bytes...
I believe the actual average size is more like 1KB to 2KB of course depending on how
many extents are in a fileInode's ExtentMap and how many elements are in a dirInode's DirMap.